index.js

```diff
@@ -11,7 +11,7 @@ var path = require("path")
 var os = require("os")
 var deepcopy = require('deepcopy')
 
-const VERSION = "0.0.20"
+const VERSION = process.env.npm_package_version
```

Comment:
this will be versioned from the _runner's_ package.json, BEWARE. You probably want to test this